Cargo.toml

```
@@ -105,7 +106,7 @@ features = [
 wayland-client = { version = "0.29.4", default_features = false,  features = ["use_system_lib"], optional = true }
 wayland-protocols = { version = "0.29.4", features = [ "staging_protocols"], optional = true }
 sctk = { package = "smithay-client-toolkit", version = "0.16.0", default_features = false, features = ["calloop"],  optional = true }
-sctk-adwaita = { version = "0.4.1", optional = true }
+sctk-adwaita = { version = "0.5.1", optional = true }
```

`ab_glyph` [is sctk-adwaita default](https://github.com/PolyMeilex/sctk-adwaita/blob/master/Cargo.toml#L23) so you'll want to disable that here to allow "notitle" to work & for crossfont users to avoid ab_glyph in the dependency tree.
```suggestion
sctk-adwaita = { version = "0.5.1", default_features = false, optional = true }
```

---

That makes sense, I had a wrong memory that it's not default. Thx.

---

Done!